test/valid-data/type-mapped-undefined-value/schema.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$ref":        "#/definitions/MyObject",
+    "$schema":     "http://json-schema.org/draft-07/schema#",
+    "definitions": {
+        "MyObject": {
+            "type": "object",
+            "additionalProperties": {
```

Comment:
Could we omit this entirely?

Reply:
Yes, i think we could

Reply:
I didn't mean the test. I meant the `additionalProperties` property.

Reply:
Sorry, my bad. Can you clarify for me what exactly you meant by omit additionalProperties property. We could change it to false value instead of object.

Reply:
![image](https://user-images.githubusercontent.com/40061451/114634629-a2e2d180-9ccb-11eb-9950-87a6106fbffa.png)
We could do it in ObjectType

Reply:
![image](https://user-images.githubusercontent.com/40061451/114634999-78454880-9ccc-11eb-819a-283ff7489e70.png)

![image](https://user-images.githubusercontent.com/40061451/114635040-8abf8200-9ccc-11eb-9a9e-3c403075bd86.png)

Reply:
Hmm, that's not correct, no? `never` would mean we don't allow additional properties but `unknown` means the same as `any`. What do we do for `any`? We should do the same for `unknown`.

Reply:
I'm not sure I understand what you are talking about. It was about `undefined`, not `unknown`. `undefined` means the same as `never` in that case

Reply:
My fault. I misread `Record<string, undefined>` as `Record<string, unknown>`